src/python_tools/notebooks/python_tools_usage_examples.py

```diff
@@ -0,0 +1,84 @@
+# ---
+# jupyter:
+#   jupytext:
+#     formats: ipynb,py:light
+#     text_representation:
+#       extension: .py
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.10.1
+#   kernelspec:
+#     display_name: Python 3
+#     language: python
+#     name: python3
+# ---
+
+# # Python Tools Usage Examples
+
+from software.proto.sensor_msg_pb2 import SensorProto
+import numpy as np
+import matplotlib.pyplot as plt
+import os
+from python_tools.proto_log import ProtoLog
+from python_tools.sensor_proto_log import SensorProtoLog
+from shared.proto.tbots_software_msgs_pb2 import PrimitiveSet
+
+# +
+# Create a ProtoLog from the chunked Protobuf message files in the directory,
+# with contained message type PrimitiveSet
+primitive_set_proto_log = ProtoLog(
+    "/home/jordan/protologgertest/PrimitiveSet/", PrimitiveSet
+)
+
+# create a list of primitives for robot 0 from the primitive sets
+robot_0_primitives = [
+    primitive_set.robot_primitives[0] for primitive_set in primitive_set_proto_log
+]
+plt.figure(figsize=(20, 5))
+
+# build a list of autochip move final angles and their corresponding timestamps from the list of robot 0 primitives
+final_angles = []
+timestamps = []
+for idx, prim in enumerate(robot_0_primitives):
+    if prim.HasField("autochip_move"):
+        final_angles.append(prim.autochip_move.final_angle.radians)
+        timestamps.append(
+            primitive_set_proto_log[idx].time_sent.epoch_timestamp_seconds
+        )
+
+# plot the result
+plt.figure(figsize=(10, 5))
+plt.scatter(timestamps, final_angles)
+plt.title("Autochip Move final angles for robot 0")
+# -
+
+sensor_proto_log = SensorProtoLog("/home/jordan/protologgertest/SensorProto/")
```

Comment:
Is there a "cleaner" way to set the directory? At least some comment to point here that the path should be changed  (even though I think it's obvious, seeing a placeholder instead of "/home/jordan" might be better for the codebase)

Reply:
None of the notebooks were changed except for converting from ipynb to py. 

Opened https://github.com/UBC-Thunderbots/Software/issues/1951 for sample data in the notebooks directory